data/org.freedesktop.portal.ScreenCast.xml

```diff
@@ -192,6 +192,13 @@
               stream.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>source_type (u)</term>
```

Comment:
I don't think it should need to be a tuple. The other ones are because they contain multiple values.

Reply:
I didn't realize that (u) is a structure or a touple.